docs/adr/0031-flow-image-support.md

```diff
@@ -0,0 +1,54 @@
+---
+status: "draft"
+---
+# Ability to bulk create flow segments
+
+## Context and Problem Statement
+
+Currently the TAMS API has three flow type: Video, Audio and Data.  There are a number of use cases around storing images on the flow timeline to represent where they have been extracted from the content.  These include UI elements, content navigation and VOD workflows.
+
+Currently the nearest option is to treat these as non-continuous video and use the existing flow data model.  This has the issue that the frame rate field is mandatory but does not make sense for still images.  For a regular stream of images (eg 1 per segment) then it could be argued to represent this as 1 over the segment length, however this implies that the image is relevant for the duration of the segment rather than representing a point in time.  Additionally if the images are not evenly spaced (eg shot changes) then this does not make sense.
```

Comment:
`frame_rate` isn't a mandatory property for video Flows but I kind of agree that the expectation of still images is different from moving images / video.

I imagine that a still image that corresponds to a video frame has a segment timerange matching the associated video frame, e.g. segment timerange is `[50:0_50:20000000)`. A still image can also be instantaneous to support use in compositions with any frame rate or in non-video uses, e.g. segment timerange is `[0:0]`.